source/less/blocks/price.less

```
@@ -1,7 +1,155 @@
 .price {
   background-color: @color-smoky-white;
+  position: relative;
+  padding-top: 102px;
+  padding-bottom: 40px;
+  overflow: hidden;
+  min-height: 480px;
+
+  &::before {
+    content: "";
+    position: absolute;
+    top: 0;
+    border-top: 20px solid @color-white;
+    border-right: 100vw solid @color-smoky-white;
+  }
+}
+
+.price__box {
+  width: calc(100% * 4 - 160px);
+  margin-bottom: 40px;
+}
```

Красоты ради можно добавить плавный переход)
```suggestion
.price__box {
  width: calc(100% * 4 - 160px);
  margin-bottom: 40px;
  transition: transform 0.3s ease-out;
}
```